library/alloc/src/vec.rs

```diff
@@ -3026,7 +3026,10 @@ impl<T> Drain<'_, T> {
     }
 }
 
-/// An iterator produced by calling `drain_filter` on Vec.
+/// A draining iterator with filter predicate for `Vec`.
```

Comment:
This doesn't seem to say anything the name of the method doesn't ... how about reusing the description from `drain_filter`?
```suggestion
/// An iterator which uses a closure to determine if an element should be removed.
```

Reply:
This does seemed better but it does not clearly shows that the iterator is draining.

Reply:
Well it does have `Drain` in the name 😄 I think it's close enough and better to be consistent.